src/lib.pro

```diff
@@ -18,19 +18,25 @@ public_headers += \
 include(src.pri)
 
 win32: target.path = $${PREFIX}/bin
-else: target.path = $${PREFIX}/lib
+else: target.path = $$[QT_INSTALL_LIBS]
 INSTALLS += target
```

Comment:
@martyone Have you tried building qmllive for the sdk after this?

Now `$$[QT_INSTALL_LIBS]` gets to qmllive-install by relative path, so there is no lib folder

Reply:
Indeed, it should have been `$${PREFIX}/$$[QT_INSTALL_LIBS]` I believe. @mlehtima could you please fix that?

Reply:
$${PREFIX} is already included in $$[QT_INSTALL_LIBS]

Reply:
During qmllive build for the sdk, a custom prefix is set:
https://github.com/sailfishos/sdk-build-tools/blob/master/buildqmllive.sh#L236

Reply:
PR opened #8